Bug in release 22.1.0 "[object Object] error Command failed with exit code 1."

Which version of electron-builder are you using?
* **Version**: 22.1.0

Which version of electron-updater are you using (if applicable)?
* **Version**: 4.2.0

What target are you building for?
* **Target**: nsis-web

<!-- Enter your issue details below this comment. -->
* **package.json**:
```
    "build": {
        "appId": "com.xxx.yyy",
        "productName": "Viewer",
        "afterSign": "crv/config/afterSignHook.js",
        "directories": {
            "output": "build"
        },
        "nsisWeb": {
            "artifactName": "${productName}-Setup-${version}.${ext}",
            "oneClick": true,
            "perMachine": false,
            "allowToChangeInstallationDirectory": false,
            "runAfterFinish": true,
            "deleteAppDataOnUninstall": true,
            "differentialPackage": true
        },
        "publish": {
            "provider": "generic",
            "url": "https://www.xxxxx.com/${os}",
            "channel": "latest"
        },
        "files": [
            "!apps${/*}",
            "!tsconfig.json",
            "!tslint.json",
            "!yarn-error.log",
            "!yarn.lock"
        ],
        "win": {
            "extraResources": [
                {
                    "from": "crv/assets/data",
                    "to": "../",
                    "filter": "test*"
                }
            ],
            "icon": "crv/assets/icons/icon.ico",
            "target": [
                {
                    "target": "nsis-web",
                    "arch": [
                        "x64",
                        "ia32"
                    ]
                }
            ]
        },
        "mac": {
            "extraResources": [
                {
                    "from": "crv/assets/data",
                    "to": "../../../",
                    "filter": "test*"
                }
            ],
            "icon": "crv/assets/icons/icon.icns",
            "hardenedRuntime": true,
            "gatekeeperAssess": false
        }
    },
```

* **Error**:
```
$ electron-builder .
  • electron-builder  version=22.1.0 os=10.0.18362
  • loaded configuration  file=package.json ("build" field)
  • electron-rebuild not required if you use electron-builder, please consider to remove excess dependency from devDependencies

To ensure your native dependencies are always matched electron version, simply add script `"postinstall": "electron-builder install-app-deps" to your `package.json`
  • writing effective config  file=build\builder-effective-config.yaml
  • rebuilding native dependencies  dependencies=sqlite3@4.1.0 platform=win32 arch=x64
  • rebuilding native dependency  name=sqlite3 version=4.1.0
  • packaging       platform=win32 arch=x64 electron=7.1.1 appOutDir=build\win-unpacked
  • rebuilding native dependencies  dependencies=sqlite3@4.1.0 platform=win32 arch=ia32
  • rebuilding native dependency  name=sqlite3 version=4.1.0
  • packaging       platform=win32 arch=ia32 electron=7.1.1 appOutDir=build\win-ia32-unpacked
  • building        target=nsis-web file=build\nsis-web\Viewer-Setup-1.0.18.exe archs=x64, ia32 oneClick=true perMachine=false
  • building embedded block map  file=build\nsis-web\crv-1.0.18-ia32.nsis.7z
  • building embedded block map  file=build\nsis-web\crv-1.0.18-x64.nsis.7z
  ⨯ [object Object]
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "run-electron-builder" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

With electron-builder version 21.2.0 everything works fine.
<!-- If you want, you can donate to increase issue priority (https://www.electron.build/donate) -->

I checked and **can not** confirm that the icon size causes the problem. Probably you built your executable with the option `nsis `as follow:

```
    "build": {
        "nsis": {
            ...
        },
        "win": {
            "target": [
                {
                    "target": "nsis",
                    "arch": [
                        "ia32",
                        "x64"
                    ]
                }
            ]
        },
    },
```
In my solution I use the [web-installer](https://www.electron.build/configuration/nsis#web-installer). As describe in the documentation this is the better way to go if you file is huge, because the web Installer automatically detects OS architecture and downloads corresponding package file which is smaller by half than the nsis package. When I build with `nsis `option everything goes through. 

With the debug option `set DEBUG="electron-builder"` I get the following error:
```
Processing config: C:\Users\Krzysztof\AppData\Local\electron-builder\Cache\nsis\nsis-3.0.4\nsisconf.nsh
Processing script file: "<stdin>" (ACP)
!insertmacro: macro "_If" requires 4 parameter(s), passed 6!
Error in macro downloadApplicationFiles on macroline 19
Error in macro installApplicationFiles on macroline 63
!include: error in script: "installSection.nsh" on line 60
Error in script "<stdin>" on line 160 -- aborting creation process
  • exited          command=makensis.exe code=1 pid=9920
  • async task error  error=
                        Error: C:\Users\Krzysztof\AppData\Local\electron-builder\Cache\nsis\nsis-3.0.4\Bin\makensis.exe exited with code ERR_ELECTRON_BUILDER_CANNOT_EXECUTE
                            at ChildProcess.childProcess.once.code (C:\ZNF_SandBox2\Mvc\StorkCMS\Stork.SPA\node_modules\app-builder-lib\node_modules\builder-util\src\util.ts:239:14)
                            at Object.onceWrapper (events.js:277:13)
                            at ChildProcess.emit (events.js:189:13)
                            at maybeClose (internal/child_process.js:970:16)
                            at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
  ⨯ [object Object]
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! crv@1.0.1 electron-builder: `electron-builder .`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the crv@1.0.1 electron-builder script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Krzysztof\AppData\Roaming\npm-cache\_logs\2019-11-18T05_49_48_379Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

In my opinion this is a bug.

--- a/packages/app-builder-lib/templates/nsis/include/webPackage.nsh
+++ b/packages/app-builder-lib/templates/nsis/include/webPackage.nsh
@@ -17,7 +17,7 @@
 	          StrCpy $packageUrl "$packageUrl/${APP_32_NAME}"
 	        ${endif}
 		    !else
-	        ${if} ${IsNativeAMD64} == true
+	        ${if} ${IsNativeAMD64}
 	          StrCpy $packageUrl "$packageUrl/${APP_64_NAME}"
 	        ${else}
 	          StrCpy $packageUrl "$packageUrl/${APP_32_NAME}"
@@ -31,7 +31,7 @@
     !endif
   !endif
 
-  ${if} ${IsNativeARM64} 
+  ${if} ${IsNativeARM64}
     StrCpy $packageArch "ARM64"
   ${elseif} ${IsNativeAMD64}
     StrCpy $packageArch "64"

